Objects/codeobject.c

```diff
@@ -241,8 +241,12 @@ PyCode_NewWithPosOnlyArgs(int argcount, int posonlyargcount, int kwonlyargcount,
     co->co_nlocals = nlocals;
     co->co_stacksize = stacksize;
     co->co_flags = flags;
+    assert(PyBytes_GET_SIZE(code) <= INT_MAX);
```

Comment:
We already check for this in the beginning of the function;
https://github.com/python/cpython/blob/18f41c04ff4161531f4d08631059fd3ed37c0218/Objects/codeobject.c#L211-L217